homeassistant/components/shelly/const.py

```diff
@@ -72,6 +72,18 @@
     "button3": 3,
 }
 
+RGB_EFFECTS = {
+    0: "Off",
+    1: "Meteor shower",
+    2: "Gradual change",
+    3: "Breath",
+    4: "Flash",
```

Comment:
Bulb Duo (`SHCB-1`) and RGBW2 (`SHRGBW2`) shows the following effects on web page:
![image](https://user-images.githubusercontent.com/1858925/111544601-629c3c00-877d-11eb-85dc-1f6b40d1aae6.png)
And also Shelly Color Bulb (`SHBLB-1`):
![image](https://user-images.githubusercontent.com/1858925/111544711-8c556300-877d-11eb-8ab7-5fa09f1d64c5.png)
https://shelly-api-docs.shelly.cloud/#shelly-bulb-overview
So I think we should split into two sets.
Also to be aligned with other places in the integration and with the manufacture names we should use capital first letters (`Shower`, `Change`)

Reply:
Order is fixed in latest firmware ;-)